allensdk/brain_observatory/ecephys/stimulus_analysis/__main__.py

```diff
@@ -72,12 +79,25 @@ def add_metrics_to_units_table(nwb_path, stimulus_classes, args):
 
     print(nwb_path)
 
-    session = EcephysSession.from_nwb_path(nwb_path)
+    try:
+        session = EcephysSession.from_nwb_path(nwb_path)
+    except FileNotFoundError:
+        return pd.DataFrame()
 
     metrics = [stim(session, params=args).metrics for stim in stimulus_classes]
-    metrics.insert(0, session.units)
+    units_table = session.units
+    units_table['nwb_file'] = nwb_path
+    metrics.insert(0, units_table)
+
+    final_table = reduce(lambda left,right: pd.merge(left, right, on='unit_id'), metrics)
+
+    fname = os.path.basename(nwb_path)
+    fname = fname.split('.')[0] + '.stimulus_analysis.csv'
+
+    output_file = os.path.join('/mnt/nvme0/unit_tables', fname)
```

Comment:
what?

Reply:
Sorry...this file is definitely not production-ready. This is what I've been using to generate units tables locally, but it needs to be cleaned up before merging.

Reply:
I'll go ahead and remove this for now and we can add it back later